### ISSUE TITLE
[stretch goal] Connect with people

### ISSUE THREAD

**Issue by plexus** [GaiwanTeam/compass#9]
**This is a stretch goal ticket, it's nice to have stuff that we'll start on once all the basic functionality is in place.**

Conferences are all about connecting with people, we want to make it easy to keep track of who you met, and to keep in touch with them. For this we're thinking of having a QR based system, so that if you're standing next to a person they can show you the PR in their compass app, and you can then request to connect with them. (This will probably be the only way, so you can only connect when you're physically next to each other).

So you'll end up with a sort of "friends" list (although I'm not sure about calling them friends...). 

This should plug into privacy controls, e.g. people may show their real name, email, etc only to connections. 

You should be able to add a personal note to connections ("chatted at the bar about rama").

The plan is to email people a list of their connections, with the notes and any contact details the connections chose to share.
